README.md

```diff
@@ -519,3 +519,176 @@ You can omit this argument to ignore this filter.
 
 This is the `reference` you gave at the time of sending the notification. The `reference` can be a unique identifier for the notification or an identifier for a batch of notifications.
 
+## Get a template by ID
+
+```
+response = notifications_client.get_template(
+    'template_id'
+)
+```
+
+<details>
+<summary>
+Response
+</summary>
+
+If the request is successful, `response` will be a `dict`:
+
+```python
+{
+    "id": "notify_id", # required
+    "type": "sms" | "letter" | "email", # required
+    "created_at": "created at", # required
+    "updated_at": "updated at", # required
+    "version": "version", # required
+    "created_by": "created by", # required
+    "body": "Body of the notification", # required
+    "subject": "Subject of an email notification of None if an sms message"
+}
+```
+
+Otherwise the client will raise a `HTTPError`:
+<table>
+<thead>
+<tr>
+<th>`error.status_code`</th>
+<th>`error.message`</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>
+<pre>404</pre>
+</td>
+<td>
+<pre>
+[{
+    "error": "NoResultFound",
+    "message": "No result found"
+]}
+</pre>
+</td>
+</tr>
+</tbody>
+</table>
+</details>
+
+## Get a template by ID and version
+
+```
+response = notifications_client.get_template_version(
+    'template_id',
+    'version'
```

Comment:
version is an integer, not a string... we should think about how to say that here. it's never been a problem with our str ids before i guess